### PR TITLE
bump google analytics version

### PR DESCRIPTION
**Pull request by cgardens**

### CODE REVIEW

File: tools/bin/ci_credentials.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ jq --arg v "$GH_INTEGRATION_TEST_CREDS" '.access_token = $v' airbyte-integration
 
 mkdir airbyte-integrations/connectors/source-googleanalytics-singer/secrets/
 echo "$GOOGLE_ANALYTICS_TEST_CREDS" > airbyte-integrations/connectors/source-googleanalytics-singer/secrets/config.json
-echo "GOOGLE_ANALYTICS_TEST_TRACKING_ID" > airbyte-integrations/connectors/source-googleanalytics-singer/secrets/tracker.txt
```
Comment (cgardens) [inline tools/bin/ci_credentials.sh:17]: @jrhizor was this intentional?

**Review by michel-tricot — APPROVED** [review]

File: tools/bin/ci_credentials.sh (hunk 1 above)
Comment (jrhizor) [reply tools/bin/ci_credentials.sh:17]: Yeah the tracker id is used for the source test but nothing else because it's used to trigger a pageview.

Comment (jrhizor) [reply tools/bin/ci_credentials.sh:17]: Oh, didn't see the `$`. Definitely not intentional. I guess it was "working" by piggybacking off of old page views.

Comment (jrhizor) [reply tools/bin/ci_credentials.sh:17]: Definitely shouldn't impact the functionality of the GA integration itself.

**Review by jrhizor — APPROVED** [review]

**Pull request merged**